README.md

```diff
@@ -8,3 +8,37 @@ Install it from [npm](https://www.npmjs.com/package/nodejs_steambot) or check ou
 # Support
 
 Report bugs on the [issue tracker](https://github.com/jacobtruman/nodejs_steambot/issues).
+
+If you need help extracting your steam file, contact [LeeTheGayKid](http://www.steamcommunity.com/id/jingyong) <b>You will need your phone to be rooted !</b>
+
+# Installation
+
+This will guide users on how to install the bot
+
+1. Download and extract the bot to somewhere easy to access.
+
+E.g Desktop
+
+2. Run cmd and cd to the file  
+
+E.g (my bot's file is named bot) `cd desktop` then, `cd bot`
+
+3. Do `npm install`
+4. After it is done, make a subfolder named `configs`
```

Comment:
I should have mentioned before that I moved away from the README in favor of the wiki, but never finished it. All of what you submit here that is approved, I will put on the wiki instead of the README.